tasks/release/ReleaseVersionController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace MailPoetTasks\Release;
+
+require_once __DIR__ . '/Jira.php';
+
+class ReleaseVersionController {
+
+  /** @var Jira */
+  private $jira;
+
+  /** @var string */
+  private $project;
+
+  function __construct(Jira $jira, $project) {
+    $this->jira = $jira;
+    $this->project = $project;
+  }
+
+  static function createWithJiraCredentials($token, $user, $project) {
+    return new self(new Jira($token, $user, $project), $project);
+  }
+
+  function assignVersionToCompletedTickets($version = null) {
+    $version = $this->checkVersion($version);
+    if (!$version) {
+      throw new \Exception('The version is invalid or already released');
+    }
+
+    $output = [];
+    $output[] = "Setting version $version to completed tickets in $this->project...";
+    $issues = $this->getDoneIssuesWithoutVersion();
+    $result = array_map(function ($issue) use ($version) {
+      return $this->setIssueFixVersion($issue['key'], $version);
+    }, $issues);
+    $output[] = "Done, issues processed: " . count($result);
+
+    return join("\n", $output);
+  }
+
+  private function getDoneIssuesWithoutVersion() {
+    $jql = "project = $this->project AND status = Done AND (fixVersion = EMPTY OR fixVersion IN unreleasedVersions()) AND updated >= -52w";
```

Comment:
Do we need to update also issues which already have some version assigned (`OR fixVersion IN unreleasedVersions()`)? What if we plan 2 releases upfront and manually set version in Jira. Also method name says `IssuesWithoutVersion`

Reply:
I don't remember planning several releases at once, there is a more common case when we plan to release a patch version then decide it should be a minor version, so we should update it for all already tagged tickets. Though we can just rename it in Jira, so I'll remove this. 

If this ever becomes necessary we can put it back with an `--update` CLI option.